service/src/test/kotlin/org/opensearch/simpleschema/rest/CreateDomainIT.kt

```diff
@@ -16,7 +16,7 @@ class CreateDomainIT : PluginRestTestCase() {
         )
         val id = createResponse.get("objectId")
         Assert.assertNotNull("Id should be present", id)
-        Thread.sleep(100)
+        Thread.sleep(200)
```

Comment:
For the general `Thread.sleep` changes, I'm starting to get concerned about their frequency. What are we waiting for with the sleeps, and is there any way to handle it better? Is it to avoid concurrency issues?

Reply:
waiting for the server to correctly load and send the sync refresh between cluster nodes